ModelViewer/ModelViewer/ViewerGraphicsWindow.h

```diff
@@ -16,13 +16,19 @@ class ViewerGraphicsWindow : public OpenGLWindow
     void resetView();
 
     bool loadModel(QString filepath = QString());
+
+    bool loadVertexShader(QString vertfilepath = QString());
+    bool loadFragmentShader(QString fragfilepath = QString());
+
     bool addPrimitive(QString filepath);
 
     // Mouse variables
     QMatrix4x4 sceneMatrix;
     float viewportX = 0;
     float viewportY = 0;
 
+    QMatrix4x4 modelview;
```

Comment:
I'm not a fan of the duplicated code related to the modelView matrix, but it's okay for now.  Once Jonathan reworks the mouse controls, we will have separate rotation, translation, and scale matrices which can be used to compute the modelView when needed.

Reply:
agreed. I just wasn't sure how to go from modelviewprojection matrix back to modelview matrix